tmk_core/common/chibios/eeprom_stm32.h

```diff
@@ -52,7 +52,7 @@
 
 #ifndef EEPROM_START_ADDRESS
     #if defined (MCU_STM32F103RB)
-        #define FEE_MCU_FLASH_SIZE  128     // Size in Kb
+        #define FEE_MCU_FLASH_SIZE  64     // Size in Kb
```

Comment:
Did we want to chang this?

Reply:
Yes - this was the bug.
128 sometimes works due to chip binning - you can receive a chip that has access to 128k just because the chip was sold as a lower one. But officially, it only has 64k.

Reply:
Ah, okay.  That makes sense, and I just wanted to make sure!